galvan-support/src/main/java/org/terracotta/testing/rules/BasicExternalCluster.java

```diff
@@ -108,7 +108,19 @@ public BasicExternalCluster(File clusterDirectory, int stripeSize, List<File> se
 
   @Override
   public Statement apply(Statement base, Description description) {
```

Comment:
Right... this PR changes the content of the displayName string so that those paths are a little shorter.  Eg:
```
[cdennis@Chriss-MacBook-Pro galvan-support (bec-display-name *)]$ ls target/cluster/
BasicExternalClusterActivePassiveIT3211422405588413478						terminateAllServers(BasicExternalClusterImmediateControlIT)4674232810595424681
BasicExternalClusterClassRuleIT5121380554379762459						terminateOnePassive(BasicExternalClusterImmediateControlIT)4622088741602320130
CorruptConfigWithClassRuleIT3055325380871200805							testConnectionViaURI(BasicExternalClusterMethodRuleIT)123431684609481913
InvalidPathTestIT4645686398243518311								testDirectConnection(BasicExternalClusterMethodRuleIT)3553423029669824777
startAllServers(BasicExternalClusterImmediateControlIT)759001885528126494			waitForActive(BasicExternalClusterImmediateControlIT)113079066240505490
startOneServer(BasicExternalClusterImmediateControlIT)8362287716628713182			waitForRunningPassivesInStandby(BasicExternalClusterImmediateControlIT)5303871664680397283
terminateActive(BasicExternalClusterImmediateControlIT)3322173363172794770
```